### PR TITLE
SPY-11630 All format in one project

### PR DESCRIPTION
**Pull request by fernandpeng**
Merged all format into one

ObjC & Swift both have a separated Workspace with different integration methods.

### CODE REVIEW

File: Swift/Direct integration/Direct integration/Banner & MPU/BannerViewController.swift [hunk 1]
```diff
@@ -1,85 +1,82 @@
 //
-//  ViewController.swift
-//  ThumbnailExample
+//  BannerManager.swift
+//  Direct Integration Sample
 //
-//  Copyright © 2020 Ogury Co. All rights reserved.
+//  Created by Fernand Peng on 09/11/2021.
```
Comment (Pernic) [inline Swift/Direct integration/Direct integration/Banner & MPU/BannerViewController.swift]: remark (blocking) : remove all 
`Created by Fernand Peng on 09/11/2021.`
on the top of file, you can write : 
` Copyright © 2020 Ogury Co. All rights reserved.`

File: Swift/Direct integration/Direct integration/Banner & MPU/BannerViewController.swift [hunk 2]
```diff
@@ -1,85 +1,82 @@
 //
-//  ViewController.swift
-//  ThumbnailExample
+//  BannerManager.swift
```
Comment (Pernic) [inline Swift/Direct integration/Direct integration/Banner & MPU/BannerViewController.swift]: remark(no-blocking): wrong name of file

File: Swift/Direct integration/Direct integration/Banner & MPU/BannerViewController.swift [hunk 3]
```diff
@@ -1,85 +1,82 @@
...
     var smallBanner: OguryAdsBanner?
     var mpuBanner: OguryAdsBanner?
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        self.addNewStatus("Choice Manager Loading...")
+        self.addNewStatus("[Choice Manager] Loading...")
         
         //The setup of Ogury Choice Manager and Ogury Ads is done AppDelegate.swift file.
         OguryChoiceManager.shared().ask(with: self) { (error, answer) in
```
Comment (Pernic) [inline Swift/Direct integration/Direct integration/Banner & MPU/BannerViewController.swift]: suggestion (no-blocking) : put Choice Manager on a different file

File: Swift/Direct integration/Direct integration/Banner & MPU/BannerViewController.swift [hunk 4]
```diff
@@ -1,85 +1,82 @@
...
+                self.addNewStatus("[Choice Manager] error : \(error.debugDescription)")
             }
         }
     }
 
     @IBAction func loadMPUBtnPressed(_ sender: Any) {
-        self.addNewStatus("MPU Loading ...")
-        mpuBanner = OguryAdsBanner.init(adUnitID: "ogury_adunit")
+        addNewStatus("[Ad][MPU] Loading ...")
+        mpuBanner = OguryAdsBanner.init(adUnitID: ConstantKeys.mpuAdUnit)
```
Comment (Pernic) [inline Swift/Direct integration/Direct integration/Banner & MPU/BannerViewController.swift]:
```suggestion
        mpuBanner = OguryAdsBanner(adUnitID: ConstantKeys.mpuAdUnit)
```

Comment (Pernic) [reply Swift/Direct integration/Direct integration/Banner & MPU/BannerViewController.swift]: .init is not needed in Swift

**Review by Pernic — CHANGES_REQUESTED** [review]
I don't have finish to read all the PR, but it's it's pretty good. I have make just small remark 👍

File: Swift/Direct integration/Direct integration/Banner & MPU/BannerViewController.swift (hunk 4 above)
Comment (fernandpeng) [reply Swift/Direct integration/Direct integration/Banner & MPU/BannerViewController.swift]: Updated in 750d11cae96c04f3272b4a34935c4ea6ed56a689

File: Swift/Direct integration/Direct integration/Banner & MPU/BannerViewController.swift (hunk 3 above)
Comment (fernandpeng) [reply Swift/Direct integration/Direct integration/Banner & MPU/BannerViewController.swift]: I think it's better to keep it in every ViewController so all of them have the whole logic complete example. (beside setup is done in AppDelegate and Constant store in separated file)

File: Swift/Direct integration/Direct integration/Banner & MPU/BannerViewController.swift (hunk 1 above)
Comment (fernandpeng) [reply Swift/Direct integration/Direct integration/Banner & MPU/BannerViewController.swift]: Updated in 750d11cae96c04f3272b4a34935c4ea6ed56a689

File: Swift/Direct integration/Direct integration/Banner & MPU/BannerViewController.swift (hunk 2 above)
Comment (fernandpeng) [reply Swift/Direct integration/Direct integration/Banner & MPU/BannerViewController.swift]: Updated in 750d11cae96c04f3272b4a34935c4ea6ed56a689

File: .gitignore [hunk 5]
```diff
@@ -0,0 +1,5 @@
+*/Pods/
```
Comment (Pernic) [inline .gitignore:1]: remark(no-blocking) : you can use some tools to generate the gitignore : https://www.toptal.com/developers/gitignore

**Review by Pernic — DISMISSED** [review]

File: .gitignore (hunk 5 above)
Comment (fernandpeng) [reply .gitignore:1]: I used this website as reference actually, but as the projet organised differently(there is ObjC and Swift as parent folder + one share workspace for Admob and direct integration) needed to customise a little bit :).

I will do the update for the ObjC version & Readme before merge this PR,
thanks in advance :)

**Review by Pernic — COMMENTED** [review]
The UI is not optimized for small iPhone.

**Comment by fernandpeng** [comment]
> The UI is not optimized for small iPhone.

fixed in 75a4bd7 - tested on iPod touch - 7th generation

**Comment by Pernic** [comment]
MPU have not the good size :

https://user-images.githubusercontent.com/33022315/182330305-ef68047b-38b0-448f-81a5-d2120a68252d.mov

**Comment by fernandpeng** [comment]
> MPU have not the good size :
> 
>  Screen.Recording.2022-08-02.at.10.32.46.mov

Fixed in d5280dd

**Comment by fernandpeng** [comment]
@Pernic Applied feedbacks in d5280dd
and also updated admob integration in e722cd8

**Review by Pernic — APPROVED** [review]

**Pull request merged**